impeller/geometry/path.h

```diff
@@ -34,6 +34,13 @@ enum class FillType {
   kAbsGeqTwo,
 };
 
+enum class Convexity {
+  kUnknown,
+  kConvex,
+  kConcave,
+  kComplex,
```

Comment:
Document these - I'm not sure what you mean by complex (holes? multiple non-overlapping polygons with different convexity? self-intersection?)

Reply:
complex polygons have self intersection. I think for now since I'm not using these other cases, I'll drop it down to just unknown and convex